Rakefile

```diff
@@ -12,7 +12,15 @@ task :asset_compilation_environment do
   ENV['SECRET_TOKEN'] = 'foo'
   ENV['GITHUB_TOKEN'] = 'foo'
   ENV['PRECOMPILE'] = '1'
-  ENV['DATABASE_URL'] = 'mysql2://none@none/none'
+
+  config = Rails.application.config
+  def config.database_configuration
+    {}
+  end
+
+  ar = ActiveRecord::Base
+  def ar.establish_connection
+  end
```

Comment:
migrations don't use asset_compilation_environment ... but it would blow up when doing `rake assets:compile db:migrate` ... hopefully nobody does that :D